### DISCUSSION TITLE
Front End Edit - empty fields not displaying

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3622] [Feature Requests]
I've noticed that empty Entry fields, aren't displaying the Edit icon, in the Front End Edit mode.
This seems to apply to auto-displayed Edit icons, as well as manually added ones using the {if frontedit} markup.

EE 7.3.6
PHP 8.1

In this example, Stock does have a value, but the Storage field is empty.

```
{if frontedit}
	<div>
		Stock: {product-stock:frontedit}
		Storage: {product-stock-storage:frontedit}
	</div>
{/if}
```

![Screenshot 2023-07-19 at 17 17 33](https://github.com/ExpressionEngine/ExpressionEngine/assets/36453035/2640d6a0-09d4-4eb8-bbe8-ea7324ffb89b)

**Comment by intoeetive** [comment]
This is know limitation which we are not likely to change - however I'll move this to Feature Requests.

If you need to be sure the link is present, you can place it manually in the template: https://docs.expressionengine.com/latest/advanced-usage/front-end/frontend.html#customizing-the-link-location

**Comment by ic360-com** [comment]
I've managed to create the following workaround, which creates the required output:

```
{if frontedit}
	<div>
		Stock: <!-- disable frontedit -->{if !product-stock}0{if:else}{product-stock}{/if}<!-- //disable frontedit --> {frontedit_link entry_id="{entry_id}" field_name="product-stock"}
		Storage: <!-- disable frontedit -->{if !product-stock-storage}0{if:else}{product-stock-storage}{/if}<!-- //disable frontedit --> {frontedit_link entry_id="{entry_id}" field_name="product-stock-storage"}
	</div>
{/if}
```

Nowhere near as elegant, but it works.
